feat: add custom Dockerfile for Ubuntu 20 and Ubuntu 18 improvements (#195)
openvinotoolkit/docker_ci: docs/tutorials/OV_docker_VScode.md

The purpose of this tutorial is to examine [Intel® Distribution of OpenVINO™ toolkit](https://software.intel.com/openvino-toolkit) development containers.

> This tutorial will go step-by-step to demonstrate how to set up an OpenVINO™ custom development container under Visual Studio Code* to use it as your full-time development environment.

Reviewer: ```suggestion
This tutorial provides step-by-step instructions to demonstrate how to set up an OpenVINO™ custom development container under Visual Studio Code* to use it as your full-time development environment.
```

Intel® Distribution of OpenVINO™ toolkit is produced from different open source and closed source repositories and components. As well OpenVINO has a couple of dependencies which need to be present on your computer. Additionally, to install some of them, you need to have root/admin rights. This might not be desirable. Using Docker and Visual Studio Code integration represents a much cleaner way of development. 

> We prepared Dockerfiles for custom development OpenVINO™ from source based on [Ubuntu 18](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles/ubuntu18/build_custom) and [Ubuntu 20](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles/ubuntu20/build_custom). They include the following components:

Reviewer: ```suggestion
We prepared Dockerfiles for custom development of OpenVINO™ from source based on [Ubuntu 18](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles/ubuntu18/build_custom) and [Ubuntu 20](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles/ubuntu20/build_custom). They include the following components:
```

## Build OpenVINO custom development Docker
The first step, we need to build a OpenVINO™ Docker image, by default it uses Debug mode and master branch for all repositories.

> Lets go to Docker CI repository local copy and run `docker build` from `docker_ci/dockerfiles/<os>/build_custom` folder:

Reviewer: ```suggestion
Go to the Docker CI repository local copy and run `docker build` from `docker_ci/dockerfiles/<os>/build_custom` folder:
```
